lib/Account.php

```diff
@@ -150,13 +178,32 @@ public function deauthorize($clientId = null, $opts = null)
         return OAuth::deauthorize($params, $opts);
     }
 
+    /**
+     * @param array|null $params
+     * @param array|string|null $opts
+     *
+     * @throws \Stripe\Exception\ApiErrorException if the request fails
+     *
+     * @return Account The rejected account.
+     */
+    public function reject($params = null, $opts = null)
+    {
+        $url = $this->instanceUrl() . '/reject';
+        list($response, $opts) = $this->_request('post', $url, $params, $opts);
+        $this->refreshFrom($response, $opts);
+        return $this;
+    }
+    use ApiOperations\NestedResource;
```

Comment:
Minor but I'd recommend having this at the top. It's really easy to miss and it's an important concept core to those classes